cached_network_image/pubspec.yaml

```diff
@@ -8,16 +8,16 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_cache_manager: ^3.3.0
-  octo_image: ^1.0.0
+  octo_image: ^1.0.2
   cached_network_image_platform_interface: ^1.0.0
   cached_network_image_web: ^1.0.0
 
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  mocktail: ^0.1.1
-  flutter_lints: ^1.0.4
+  mocktail: ^0.3.0
```

Comment:
Does this upgrade bring any benefits?

Reply:
No, because tests don't use any breaking changes in updated mocktail. This update just verifies  that.